lib/zoonk_web/live/follow_live.ex

```diff
@@ -2,18 +2,83 @@ defmodule ZoonkWeb.FollowLive do
+    ],
+    brazil: [
+      %{name: "Bluesky", url: "https://bsky.app/profile/zoonkbr.bsky.social", icon: "tabler-brand-bluesky", handle: "@zoonkbr"},
+      %{name: "Facebook", url: "https://www.facebook.com/zoonkbr", icon: "tabler-brand-facebook", handle: "@zoonkbr"},
+      %{name: "Instagram", url: "https://www.instagram.com/zoonkbr", icon: "tabler-brand-instagram", handle: "@zoonkbr"},
+      %{name: "LinkedIn", url: "https://www.linkedin.com/company/zoonk", icon: "tabler-brand-linkedin", handle: "@zoonk"},
+      %{name: "Reddit", url: "https://www.reddit.com/r/ZoonkBrasil", icon: "tabler-brand-reddit", handle: "r/ZoonkBrasil"},
+      %{name: "Threads", url: "https://www.threads.net/@zoonkbr", icon: "tabler-brand-threads", handle: "@zoonkbr"},
+      %{name: "TikTok", url: "https://www.tiktok.com/@zoonkbr", icon: "tabler-brand-tiktok", handle: "@zoonkbr"},
+      %{name: "X", url: "https://x.com/zoonkbr", icon: "tabler-brand-x", handle: "@zoonkbr"},
+      %{name: "YouTube", url: "https://www.youtube.com/@zoonkbr", icon: "tabler-brand-youtube", handle: "@zoonkbr"}
+    ]
+  }
+
   @impl Phoenix.LiveView
   def render(assigns) do
     ~H"""
     <ZoonkWeb.AppLayout.render flash={@flash} scope={@scope}>
-      follow us placeholder
+      <div class="max-w-2xl mx-auto">
+        <div class="text-center mb-8">
+          <h1 class="text-3xl font-bold mb-4">
+            {dgettext("page_title", "Follow us")}
+          </h1>
+          <p class="text-gray-600">
+            {dgettext("default", "Stay connected with us on social media for the latest updates and news.")}
+          </p>
+        </div>
+        
+        <div class="grid gap-4 sm:grid-cols-2 lg:grid-cols-3">
+          <div :for={social <- @social_links} class="flex items-center p-4 border rounded-lg hover:bg-gray-50 transition-colors">
+            <.a 
+              href={social.url}
+              target="_blank"
+              rel="noopener noreferrer"
+              class="flex items-center space-x-3 text-gray-900 no-underline hover:no-underline w-full"
+            >
+              <.icon name={social.icon} size={:md} class="text-gray-700" />
+              <div>
+                <div class="font-medium">{social.name}</div>
+                <div class="text-sm text-gray-500">{social.handle}</div>
+              </div>
+            </.a>
+          </div>
+        </div>
+      </div>
     </ZoonkWeb.AppLayout.render>
     """
   end
 
   @impl Phoenix.LiveView
-  def mount(_params, _session, socket) do
-    socket = assign(socket, :page_title, dgettext("page_title", "Follow us"))
+  def mount(_params, session, socket) do
+    user_language = get_user_language(socket.assigns.scope, session)
+    social_links = if user_language == "pt", do: @social_media_links.brazil, else: @social_media_links.global
+    
+    socket = 
+      socket
+      |> assign(:page_title, dgettext("page_title", "Follow us"))
+      |> assign(:social_links, social_links)
```

Comment:
Don't assign those items to the socket. It just uses more memory without need. Socket is for dynamic data.

You can just call a function or, better yet, create a config file for social media links like we have for other static data like LanguageConfig